### ISSUE TITLE
Algorithm Logic

### ISSUE THREAD

**Issue by CharlesFr** [ccxt-dev/ccxt#219]
First and foremost thanks for creating and freely sharing this library.

I understand this isn't an issue, however It's the only place I could find to post my question.

I have some experience with Quantopian and there's plenty of resources out there for algorithmic "inspiration" to trade on the stock market. However, there seems to be very little regarding the crypto market, can you guys recommend a source for starting algorithms?

**Comment by kroitor** [comment]
Hi! 

Thanks for your feedback!

Basically, most of crypto exchange markets are centralized and are operating based on the same principles on which most of conventional Forex/Stock markets do. So, you can get started with Quantopian resources. Whatever rules are true to the behaviour of the price on "classic stock markets", the same rules you can apply to crypto exchanges as well. The only difference with crypto is that it can be much more volatile during the day. The algorithmic base for TA and other types of analysis is identical.

So, in short, algotrading isn't about crypto vs fiat. It is about your strategy vs the strategies of your competitors.

Or did you mean some specific crypto-related algorithms?..

**Comment by CharlesFr** [comment]
Thanks for your prompt response, that answers my question. However, I did suspect there might be some crypto-specific algos, which perhaps are better suited to handle the volatility - just looking for some inspiration.

**Comment by kroitor** [comment]
@CharlesFr if you search for *crypto*, *trading*, *strategy* and *bot* in my starred repositories, you will find more than a thousand of various strategies, ideas, docs and code examples ;)
If that answers your question, would you mind if I close this issue for now?

**Comment by CharlesFr** [comment]
Just what I need, thanks! 👍
